lms/djangoapps/student_enrollment/utils.py

```
@@ -126,3 +129,33 @@ def post_to_zapier(zap_url, data):
         to Zapier
     """
     response = requests.post(zap_url, data=data)
+
+
+def send_success_or_exception_email(email_type, content, from_address,
+                                    to_address):
+    """
+    Sends an email to the CI platform team either confirming the
+    successful enrollment of students or email that an exception occurred
+    """
+    email_connection = create_email_connection()
+    email_subject = 'Student Enrollment Successful'
+    if email_type == 'exception':
+        email_subject = 'Student Enrollment Failed'
+
+    number_of_mails_sent = send_mail(
+        email_subject,
+        content,
+        from_address, to_address,
+        fail_silently=False,
```

`fail_silently=False` raises an exception in case of an error rather than returning zero emails, no?

---

If so, it can actually be a nicer approach, as you then don't need to log it - it should just log the error on its own. Also, it really should log an error in this case, rather than the `log.info`

---

Changed so that in case of success `log.info` is used and in case of a failure `log.error`